src/core/Jalasoft.TeamUp.Resumes.Core/Validators/PostulationValidator.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Jalasoft.TeamUp.Resumes.Core.Validators
+{
+    using FluentValidation;
+    using Jalasoft.TeamUp.Resumes.Models;
+
+    public class PostulationValidator : AbstractValidator<Postulation>
+    {
+        public PostulationValidator()
+        {
+            this.RuleFor(postulation => postulation.ProjectId)
+                .NotEmpty()
+                .NotNull();
+
+            this.RuleFor(postulation => postulation.ResumeId)
+                .NotEmpty()
+                .NotNull();
+
+            this.RuleFor(postulation => postulation.ProjectName)
+                .NotEmpty()
+                .NotNull()
+                .MaximumLength(50);
+
+            this.RuleFor(postulation => postulation.ResumeName)
+                .NotEmpty()
+                .NotNull()
+                .MaximumLength(50);
+
+            this.RuleFor(postulation => postulation.State)
+                .NotEmpty()
+                .NotNull()
+                .MaximumLength(20);
+
+            this.RuleFor(postulation => postulation.Picture)
+                .NotEmpty()
+                .NotNull();
+
+            this.RuleFor(postulation => postulation.CreationDate)
+                .NotEmpty()
+                .NotNull();
+
+            this.RuleFor(postulation => postulation.LastDate)
+                .NotEmpty()
+                .NotNull();
```

Comment:
for this validations you could check https://github.com/fundev3/teamup-projects/blob/develop/src/core/Jalasoft.TeamUp.Projects.Core/Validators/InvitationValidator.cs, and see if something applied to your case, maybe postulation.Picture, ResumeName, etc